File: Java/gesbrasilburger/src/main/java/sn/brasilburger/entity/Complement.java
package sn.brasilburger.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Complement {
    private int id;
    private String nom;
    private Double prix;
    private String image;
    private Boolean etat = true;
    private CategorieComplement categorie;
    @Override
    public String toString(){
        return " Complement->[ID: " + id + ", Nom: " + nom + ", Prix: " + prix + "]";
    }
}

File: CSharp/brasilBurger/Models/Complement.cs
namespace brasilBurger.Models
{
    public enum CategorieComplement
    {
        FRITES,
        BOISSON
    }
    public class Complement
    {
        public int Id { get; set; }
        public string Nom { get; set; }
        public decimal Prix { get; set; }
        public string Image { get; set; }
        public bool Etat { get; set; }
        public CategorieComplement Categorie { get; set; }
        public ICollection<MenuComplement> MenuComplements { get; set; }
    }
}